### ISSUE TITLE
Lullabot logo/link missing hover effect

### ISSUE THREAD

**Issue by claireristow** [Lullabot/lb-ebook-poc#44]
The header Lullabot logo link is missing a hover state. It doesn't necessary need to match lullabot.com's hover effect in terms of the animation but even just changing the background color to black (#12212b) would be good! 

<img width="482" alt="Screenshot 2024-07-17 at 4 13 08 PM" src="https://github.com/user-attachments/assets/bbe5835d-eabf-43d4-81e1-908046dffef2">
